common/src/stack/config/etc/profile.d/stack_completions.sh

```diff
@@ -0,0 +1,39 @@
+_stack_completion() {
+	local loc cur opts BASE_DIR
+
+	# BASE_DIR represents the path to the directory containing
+	# all the possible commands in their directories.
+	BASE_DIR="/opt/stack/lib/python3.*/site-packages/stack/commands/"
```

Comment:
Is the **python3.*** really fine here?

Reply:
Yep.  I acknowledge it's not specific, but we never have multiple versions of python installed simultaneously in /opt/stack.

Reply:
Got it! :+1: